console/src/channel/ontology.tsx

```diff
@@ -209,6 +215,7 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
       <Menu.Item itemKey="delete" startIcon={<Icon.Delete />}>
         Delete
       </Menu.Item>
+      <Ontology.LinkAddressMenuItem />
```

Comment:
Would probably be better to make this part of the `Link` package no? Call it `Link.CopyMenuItem`.